bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/LocaleProviderImpl.java

```diff
@@ -41,6 +45,11 @@ protected synchronized void modified(Map<String, Object> config) {
         final String script = (String) config.get(SCRIPT);
         final String region = (String) config.get(REGION);
         final String variant = (String) config.get(VARIANT);
+        final String location = (String) config.get(LOCATION);
+
+        if (location != null) {
+            this.location = PointType.valueOf(location);
```

Comment:
The location is configured as a string. If the user provides an incorrect formatted string the `valueOf` method throw an `IllegalArgumentException`. What is the expectation of the modified handling? Shouldn't we unset the location (set to `null`) or keep the old value instead of stopping the modified function handling for all other values caused by the non-catch exception?